### PR TITLE
Modal fix and enhancement on generate keypair

### PR DESCRIPTION
**Pull request by Joeper214**
ref: https://skyarch.backlog.jp/view/NC-627

This PR is also related to: https://skyarch.backlog.jp/view/NC-624#comment-1137702204

Which it introduced a new bug.

### CODE REVIEW

File: frontend/src/modal.ts [hunk 1]
```diff
@@ -43,7 +45,13 @@ const modal = function (
     modal_footer.append(
       $('<button>', {class: 'btn btn-default', 'data-dismiss': 'modal', text: 'Cancel'})
     );
+    modal_body.append($("<div>", {text: message}));
+  } else if (modal_type === ModalType.prompt){
```
Comment (skyarch-networks) [inline frontend/src/modal.ts]: __[tslint]__
* missing whitespace

File: frontend/src/modal.ts [hunk 2]
```diff
@@ -43,7 +45,13 @@ const modal = function (
...
+    modal_body.append($("<div>", {html: message}));
+  } else{
```
Comment (skyarch-networks) [inline frontend/src/modal.ts]: __[tslint]__
* missing whitespace

**Comment by Joeper214** [comment]
please review and merge PR. @root-beer :bow:

**Comment by root-beer** [comment]
LGTM! Thanks. :bow:

**Comment by Joeper214** [comment]
:bow:

**Pull request merged**